### PR TITLE
Fetch changes from pr-id when triggering from prow

### PR DESCRIPTION
**Pull request by mboukhalfa**

### CODE REVIEW

File: jenkins/scripts/files/run_integration_tests.sh [hunk 1]
```diff
@@ -122,6 +122,8 @@ if [[ "${UPDATED_REPO}" != *"${REPO_ORG}/${REPO_NAME}"* ]] ||
     git config user.name "Test"
     git remote add test "${UPDATED_REPO}"
     git fetch test
+    # if triggered from prow we cannot get the ghprbAuthorRepoGitUrl then we pull the PR
+    git fetch origin pull/${PR_ID:-0}/head:"${UPDATED_BRANCH}-branch" || true
```
Comment (tuminoid) [inline jenkins/scripts/files/run_integration_tests.sh:126]:
```suggestion
    git fetch origin pull/"${PR_ID:-0}"/head:"${UPDATED_BRANCH}-branch" || true
```

**Comment by mboukhalfa** [comment]
/hold
we want to test the pipeline on this PR by changing the jjb https://gerrit.nordix.org/c/infra/cicd/+/20713

**Pull request merged**